auth.config.ts

```diff
@@ -0,0 +1,16 @@
+import GitHub from '@auth/core/providers/github';
+import { DrizzleAdapter } from '@auth/drizzle-adapter';
+
+import { db } from './src/utils/db';
+
+import type { AuthConfig } from '@auth/core';
+
+export default {
+  adapter: DrizzleAdapter(db),
+  providers: [
+    GitHub({
+      clientId: import.meta.env.GITHUB_CLIENT_ID,
+      clientSecret: import.meta.env.GITHUB_CLIENT_SECRET
+    })
+  ]
```

Comment:
```suggestion
  ],
   callbacks: {
    session({ session, user }) {
      if (session.user) {
        session.user.id = user.id;
      }
      return session;
    }
  }
```
This adds the id field to the `session.user` object. Otherwise you wont retrieve the user id on